packages/eslint-plugin/src/rules/action-rule.ts

```diff
@@ -0,0 +1,79 @@
+import { Rule } from "eslint";
+import { CallExpression, Identifier, Literal, VariableDeclarator, ArrowFunctionExpression, Node } from "estree";
+import { isIdentifier, isLiteral } from "../lib";
+
+type ActionCallExpression =
+    CallExpression
+    & { callee: Identifier, arguments: [] | [Literal] | [ArrowFunctionExpression] | [ArrowFunctionExpression, Literal] }
+type ActionVariableDeclarator = VariableDeclarator & { id: Identifier, init: ActionCallExpression }
+
+export const actionRule: Rule.RuleModule = {
+    meta: {
+        type: 'suggestion',
+        docs: {
+            description: "Add name for every action call"
+        },
+        fixable: 'code'
+    },
+    create: function (context: Rule.RuleContext): Rule.RuleListener {
+        return {
+            VariableDeclarator: d => {
+                if (!isActionVariableDeclarator(d)) return;
```

Comment:
Maybe we also have to check if the current declaration is imported from `@reatom`?

Reply:
Maybe, but I think it's not easy task (mb I am wrong, I didn't test it on files actually)

Reply:
Make sense)